web/styles/dark_theme.css

```diff
@@ -479,7 +479,8 @@
     .new-style .tab-switcher .ind-tab:not(.selected),
     select,
     .pill-container,
-    .user-status-content-wrapper {
+    .user-status-content-wrapper,
+    #custom-expiration-time-input {
```

Comment:
This need to be added because the other CSS for this input uses ID in selector and thus overrides the color property for dark theme. The other solution is to just have the color property (or maybe all) use a class instead of ID as a selector.

Reply:
Probably the best fix involves just using a CSS variable for this color rather than having long lists like this.